critiquebrainz/ws/review/views.py

```diff
@@ -350,7 +350,10 @@ def review_list_handler():
 
     user_id = Parser.uuid('uri', 'user_id', optional=True)
     # TODO: "rating" sort value is deprecated and needs to be removed.
```

Comment:
Can we combine these two comments together?

Reply:
Will do.